components/channel_header_dropdown/menu_items/leave_channel/leave_channel.test.tsx

```diff
@@ -47,10 +47,10 @@ describe('components/ChannelHeaderDropdown/MenuItem.LeaveChannel', () => {
         };
         const makeWrapper = () => shallow(<LeaveChannel {...props}/>);
 
-        props.channel.type = Constants.DM_CHANNEL;
```

Comment:
Did these error out before you changed them? Might just need to clean up our constants a bit at some point.
Nothing to change now.

Reply:
Yes - those errored out. And yes, once all code has been converted into ts, we will need to run a cleanup of things like these. Maybe a good idea to mark these with a `TechDebt` TODO?